pkg/printer/printer.go

```diff
@@ -27,6 +27,7 @@ type FailedRule struct {
 	Name               string
 	Occurrences        int
 	Suggestion         string
+	HowToFix           string
```

Comment:
I suggest to align the terminology and name it `DocumentationUrl`, what do you think?

Reply:
You are right! Will update it!